src/helpers/ingressHandler.ts

```diff
@@ -0,0 +1,31 @@
+import { InvocationContext } from '@azure/functions';
+import { CosmosDbHelper } from './cosmosDbHelper';
+import { OpenAiHelper } from './openAiHelper';
+import { SmsEgressHandler } from './smsEgressHandler';
+
+export async function ingressHandler(message: string, context: InvocationContext): Promise<void> {
+
+    const userID = (context.triggerMetadata.userProperties as UserProperties).address;
+
+    const cosmosDbHelper = new CosmosDbHelper(process.env["COSMOSDB_CONNECTION"], process.env["COSMOSDB_DATABASE"], process.env["COSMOSDB_CONTAINER"]);
+    const smsEgressHandler = new SmsEgressHandler(process.env["SERVICEBUS_CONNECTION_EGRESS"], process.env["EGRESS_QUEUE"]);
+
+    if (message === "STOP") {
+        cosmosDbHelper.deleteMessages(userID);
+        smsEgressHandler.sendSMS(userID, "<Chat has been reset>\n\nHi I'm Guppy-AI how can I help you today?");
+        return;
+    }
+
+    const messages = await cosmosDbHelper.saveMessage(userID, message, "user");
```

Comment:
Very much optional and highly subjective, but I prefer string-based enums over hard-coded strings in code.

Reply:
this is semi-forced by the OpenAI library